tree/treeplayer/inc/TTreeReaderValue.h

```diff
@@ -132,14 +132,22 @@ class TTreeReaderValue: public ROOT::Internal::TTreeReaderValueBase {
       TTreeReaderValueBase(&tr, branchname,
                            TDictionary::GetDictionary(typeid(NonConstT_t))) {}
 
+   /// Return a pointer to the value of the current entry.
+   /// Return a nullptr and print an error if no entry has been loaded yet.
+   /// The returned address is guaranteed to stay constant while a given TTree is being read from a given file.
```

Comment:
...unless the branch addresses are manipulated directly (e.g. through `TTree::SetBranchAddress()`).

Reply:
will add this to the text